lib/ruby_lsp/requests/document_symbol.rb

```diff
@@ -108,6 +111,17 @@ def visit_class(node)
 
       sig { override.params(node: SyntaxTree::Command).void }
       def visit_command(node)
+        if @test_class && node.message.value == "test"
+          argument = node.arguments.parts.first
+          test_name = argument.child_nodes.first.value
+          create_document_symbol(
+            name: test_name,
+            kind: :method,
```

Comment:
It's not _actually_ a method but this is a good enough approximation, and keeps it consistent with tests which use the traditional syntax.